sw/device/silicon_creator/rom/data/rom_testplan.hjson

```diff
@@ -134,400 +121,1008 @@
       tests: []
     }
 
-    // Functests
     {
-      name: rom_functests
-      desc: '''Run functests developed for ROM.
+      name: rom_e2e_bootstrap_enabled_requested
+      desc: '''Verify that ROM enters bootstrap when enabled in OTP and requested.
+
+            `OWNER_SW_CFG_ROM_BOOTSTRAP_EN` OTP item must be `kHardenedBoolTrue` (`0x739`).
 
-            Functests test ROM components (e.g., drivers, libraries, etc.)
-            work as intended on-chip. However, unlike when these components are
-            embedded in the ROM, functests are linked with the OTTF, and
-            run out of flash. Additionally, unlike the ROM E2E tests,
-            functests are booted by the test ROM.
+            - Apply bootstrap pin strapping.
+            - Reset the chip.
+            - Verify that the chip responds to `READ_STATUS` (`0x05`) with `0x00`.
             '''
-      tags: ["verilator", "dv", "fpga", "silicon"]
+      tags: ["rom", "verilator", "dv", "fpga", "silicon"]
```

Comment:
you will probably want to take a pass through all test points to ensure the "rom" tag is added. It looks like some are missing above.

Reply:
Done, thanks for catching this too!